src/main/java/com/example/finance/service/CategoryService.java

```diff
@@ -69,6 +71,30 @@ public CategoryDto updateCategory(UUID id, CategoryDto category) {
         return categoryMapper.toDto(savedCategory);
     }
 
+    @Transactional
+    public void transferFundsBetweenCategories(UUID userId, String fromCategoryName, String toCategoryName,
```

Comment:
We should try to have at most 3 parameters in the methods if it is possible. Maybe instead of that all parameters, provide `TransferFunds` param? :)